typescript/cli/src/deploy/warp.ts

```
@@ -813,3 +1110,153 @@ async function getWarpApplySubmitter({
+              deployedTokenRoute: deployedContracts[chain],
+            },
+          },
+          signer,
+        );
+        const actualConfig = await cosmosNativeWarpModule.read();
+        const expectedConfig = {
+          ...actualConfig,
+          remoteRouters: allRemoteChains.reduce(
+            (acc, c) => ({
+              ...acc,
+              [context.multiProvider.getDomainId(c).toString()]: {
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)